docs/tutorials/1.5/build.md

```diff
@@ -0,0 +1,50 @@
+This is a small tutorial on how to build HORNET.
+
+**Note: This tutorial assumes that you are using Ubuntu. The setup and build process may differ on other OS.**
+
+### Preparations
+
+- Install go1.16: See [golang.org](https://golang.org/doc/install) for more information.
+  ```bash
+    wget https://golang.org/dl/go1.16.linux-amd64.tar.gz
+    tar -C /usr/local -xzf go1.16.linux-amd64.tar.gz
+    export PATH=$PATH:/usr/local/go/bin
```

Comment:
this path has to be added to the `.profile` otherwise it is gone after next login